puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_proxy_plugin_matrix.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -ex
+
+# RVM Ruby environment
+. /etc/profile.d/rvm.sh
+# Use a gemset unique to each executor to enable parallel builds
+gemset=$(echo ${JOB_NAME} | cut -d/ -f1)-${EXECUTOR_NUMBER}
+rvm use ruby-${ruby}@${gemset} --create
+rvm gemset empty --force
+#gem update --no-ri --no-rdoc
+gem install bundler --no-ri --no-rdoc
+
+bundle install --retry 5
+bundle exec rake ${rake_task} TESTOPTS="-v"
```

Comment:
I don't see where this variable is defined. Should this be a parameter?

Reply:
Yup! You're right! This variable doesn't get populated from **anywhere**.

Reply:
Maybe we should start using `set -u` in our shells scripts. While converting jobs to JJB, I've come across a significant amount of dropped, ignored, or blank variables.

Reply:
@ekohl it looks like at least a couple of the smart-proxy plugins have a default rake task so the tests at least still get run for a few.

Reply:
I think it should be a parameter with a default. That way the various jobs can override it.